src/vanna/bedrock/bedrock_converse.py

```diff
@@ -0,0 +1,85 @@
+from ..base import VannaBase
+
+try:
+    import boto3
+    from botocore.exceptions import ClientError
+except ImportError:
+    raise ImportError("Please install boto3 and botocore to use Amazon Bedrock models")
+
+class Bedrock_Converse(VannaBase):
+    def __init__(self, client=None, config=None):
+        VannaBase.__init__(self, config=config)
+
+        # default parameters
+        self.temperature = 0.0
+        self.max_tokens = 500
+        
+        if client is None:
+            raise ValueError(
+                "A valid Bedrock runtime client must be provided to invoke Bedrock models"
+            )
+        else:
+            self.client = client
+        
+        if config is None:
+            raise ValueError(
+                "Config is required with model_id and inference parameters"
+            )
+        
+        if "modelId" not in config:
+            raise ValueError(
+                "config must contain a modelId to invoke"
+            )
+        else:
+            self.model = config["modelId"]
+        
+        if "temperature" in config:
+            self.temperature = config["temperature"]
+        
+        if "max_tokens" in config:
+            self.max_tokens = config["max_tokens"]
+
+    def system_message(self, message: str) -> dict:
+        return {"role": "system", "content": message}
+
+    def user_message(self, message: str) -> dict:
+        return {"role": "user", "content": message}
+
+    def assistant_message(self, message: str) -> dict:
+        return {"role": "assistant", "content": message}
+
+    def submit_prompt(self, prompt, **kwargs) -> str:
+        inference_config = {
+            "temperature": self.temperature,
+            "maxTokens": self.max_tokens
+        }
+        additional_model_fields = {
+            "top_p": 1,  # setting top_p value for nucleus sampling
+        }
+
+        system_message = None
+        no_system_prompt = []
+        for prompt_message in prompt:
+            role = prompt_message["role"]
+            if role == "system":
+                system_message = prompt_message["content"]
+            else:
+                no_system_prompt.append({"role": role, "content": prompt_message["content"]})
```

Comment:
Fixing the request syntax based on [boto docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/bedrock-runtime/client/converse.html)
```suggestion
                no_system_prompt.append({"role": role, "content":[{"text": prompt_message["content"]}]})
```